docs/theory/apportion_abundance.md

```diff
@@ -1,52 +1,70 @@
-```
 
-Biomass estimates for each $s$ ($\textrm{M}$ and $\textrm{F}$) along transect interval $k$ are summed across $\ell$ and $\alpha$ via:
+## Back-calculating abundance from kriged biomass estimates
+
+The biomass estimates for male and female fish ($s=M$ and $s=F$, respectively) along transect interval $k$ across all lengths ($\ell$) and all ages ($\alpha$) are:
 
 $$ 
-B_{\textrm{M}}^{k} = \sum_{\textrm{M}, \ell, \alpha} B_{\textrm{M}, \ell, \alpha}^{k, \textrm{aged}} +  \sum_{\textrm{M}, \ell, \alpha} B_{\textrm{M}, \ell, \alpha}^{k, \textrm{unaged}}
+B_{\textrm{M}}^{k} =
+\sum_{\ell} B_{\textrm{M}, \ell}^{k, \textrm{unaged}} +
+\sum_{\ell, \alpha} B_{\textrm{M}, \ell, \alpha}^{k, \textrm{aged}}
 \label{eq:biomass_M} \tag{1}
 $$
 
 $$
-B_{\textrm{F}}^{k} = \sum_{\textrm{F}, \ell, \alpha} B_{\textrm{F}, \ell, \alpha}^{k, \textrm{aged}} + \sum_{\textrm{F}, \ell, \alpha} B_{\textrm{F}, \ell, \alpha}^{k, \textrm{unaged}}
+B_{\textrm{F}}^{k} =
+\sum_{\ell} B_{\textrm{F}, \ell}^{k, \textrm{unaged}} +
+\sum_{\ell, \alpha} B_{\textrm{F}, \ell, \alpha}^{k, \textrm{aged}}
 \label{eq:biomass_F} \tag{2}
 $$
 
-Similarly, biomass estimates for all fish ($B^{k}$), which is inclusive of both sexed and unsexed fish, are also summed (see: <b>[kriged biomass summation for more details](apportion_biomass.md#total-biomass-apportioned-with-length-and-age)</b>).
+The biomass estimates for all fish including both sexed and unsexed fish in the transect interval $k$ is then:
+
+$$
+B^k = B_\textrm{M}^k + B_\textrm{F}^k.
+$$
 
 
-These kriged biomass estimates are then converted to sexed ($\hat{N}_{s}^{k}$) and total ($\hat{N}^{k}$) abundance by using averaged length-weight regression output ($\overline{W}(\ell)$). Consequently, $\overline{W}(\ell)$ can be defined either by using the average length-weight relationship produced or parameterizing $\overline{W}(\ell)$ with the mean length ($\bar{\ell}$). It is important to note, however, that both $\hat{N}_{s}^{k}$ and $\hat{N}^{k}$ are calculated using $\overline{W}(\ell)$ fit from <b>all</b> individuals (i.e. male, female, and unsexed).
+The estimated abundance $\hat{N}^k$ can be back-calculated from the kriged biomass estimates using an averaged length-weight relationship $\overline{W}(\ell)$ via:
 
 $$
-\hat{N}^{k} = \frac{B^{k}}{\overline{W}(\ell)}
+\hat{N}^{k} = \frac{B^{k}}{\overline{W}(\ell)},
 \label{eq:abundance} \tag{3}
 $$
 
+where $\overline{W}(\ell)$ is the length-weight regression relationship derived from the catch data. 
+
+Similarly, $\hat{\textit{NASC}^{k}}$ can be back-calculated from the estimated abundance using the averaged differential backscattering cross-section of the $i^{\text{th}}$ stratum, $\bar{\sigma}_{bs}^i$, via:
+
+$$
+\hat{\textit{NASC}^k} = \hat{N}^k \times \bar{\sigma}_\textrm{bs}^i,
+$$
+
+when the transect interval $k$ falls in stratum $i$. See [](stratification) for more information.
```

Comment:
I see that the attention admonition concerning the strata indexing was removed. Instead of removing it entirely, would it make sense to include it here since the subsequent calculations are implicitly indexed by $i$ (or in some cases explicitly such as the proportions) sans $\overline{W}(\ell)$?

Reply:
I was thinking this would be covered by the 4-section refactoring suggested in #279, and maybe in there we can clearly list out in for what variables indexing using $k$ actually implies a specific stratum $i$ due to the geographical location of the $k$ interval?